### PR TITLE
Output sediment loading variables

### PR DESCRIPTION
**Pull request by kdeloach**
Ports sediment loading variables from VB version.

Connects #39
Connects #48

### CODE REVIEW

**Comment by kdeloach** [comment]
There is one outstanding question I have about which values to include in the totals (Ref: https://github.com/WikiWatershed/gwlf-e/pull/50/files#diff-66ce80314b7c16e4e5e83ae4418432d2R716) but otherwise I think this is done.

**Comment by mmcfarland** [comment]
These numbers are looking good, even if I modify some BMP data in the comparison gms file.  Is the `Sediment` output in this version a different unit than that reported by the vb version, and is that meant to be the case?  They all appear to be multiplied by 1000 (with allowances for precision differences). For example:

| py | vb |
|---|---|
| 100302.43 | 100.3|
|1712963.42 | 1713.0
|11150.35 | 11.2

**Comment by kdeloach** [comment]
Yes. Sediment is in kg instead of tonnes, to match Nitrogen and Phosphorus.

**Comment by mmcfarland** [comment]
I see that that's exactly what's being done with `TONNE_TO_KG = 1000` for sediment.  I'm assuming Barry recommended that unit for us?

File: gwlfe/StreamBank.py [hunk 1]
```diff
@@ -113,25 +109,31 @@ def CalculateStreamBankEros(z, Y):
...
-        z.AnimalP[Y][0] += z.AnimalP[Y][i]
-
-        z.GRLostBarnN[Y][0] += z.GRLostBarnN[Y][i]
-        z.GRLostBarnP[Y][0] += z.GRLostBarnP[Y][i]
-        z.GRLostBarnFC[Y][0] += z.GRLostBarnFC[Y][i]
-        z.NGLostBarnN[Y][0] += z.NGLostBarnN[Y][i]
-        z.NGLostBarnP[Y][0] += z.NGLostBarnP[Y][i]
-        z.NGLostBarnFC[Y][0] += z.NGLostBarnFC[Y][i]
-        z.NGLostManP[Y][0] += z.NGLostManP[Y][i]
+        z.GroundNitrSum[Y] += z.GroundNitr[Y][i]
```
Comment (mmcfarland) [inline gwlfe/StreamBank.py]: Although it's a bit tricker to compare to the original code, I think making dedicated `*Sum` variables are a big improvement for reasoning about these than using the `0` index on the array, good call for the switch.

**Comment by kdeloach** [comment]
> I see that that's exactly what's being done with TONNE_TO_KG = 1000 for sediment. I'm assuming Barry recommended that unit for us?

I believe so. According to [Model Results2.docx](https://drive.google.com/a/azavea.com/file/d/0B0hs2mX4ApaLYXBPS2RwTG1JZ1FOZ2UybXpBTEdaU2VaWVdz/view), the units of measurement for Sediment should be kg/yr.

**Comment by mmcfarland** [comment]
This looks good to me, is there still another commit coming regarding a change to the elements that get totaled?

**Comment by kdeloach** [comment]
Yes, coming right up.

**Comment by kdeloach** [comment]
Updated to correctly calculate totals.

**Comment by mmcfarland** [comment]
+1 - super thorough job.  :sparkler:

File: gwlfe/WriteOutputFiles.py [hunk 2]
```diff
@@ -703,6 +712,59 @@ def WriteOutput(z):
...
+    MeanFlow = (z.AvStreamFlowSum * CM_TO_M) * (AreaTotal * HA_TO_M2)
+
+    # Find index of month with lowest mean flow.
+    LowFlowMonth = z.AvStreamFlow.tolist().index(min(z.AvStreamFlow))
+
+    # m^3/year
+    MeanLowFlow = (z.AvStreamFlow[LowFlowMonth] * CM_TO_M) * (AreaTotal * HA_TO_M2)
+
+    # m^3/second
+    MeanFlowPS = MeanFlow * 31536000
```
Comment (mmcfarland) [inline gwlfe/WriteOutputFiles.py]: If this is per second, should this be a division instead? 
It currently produces:
```
MeanFlow: 20496145.621219214 m3/yr
MeanFlowPerSecond: 646366448310769.12 m3/s
```
which doesn't seem right.

Comment (kdeloach) [inline gwlfe/WriteOutputFiles.py]: Whoops, good catch. I think this was originally `MeanFlow * YEAR_TO_SEC` which I decided was overkill. Fixing now.

**Pull request merged**